### PR TITLE
tests: check decoded logits with ground truth

### PR DESCRIPTION
**Pull request by Huguet57**
Captura de pantalla 2024-02-24 a les 14 33 39

### CODE REVIEW

**Comment by Huguet57** [comment]
Has this weird `error: Unrecognized option: 'c'` error in `ground_truth()`, though. However, it doesn't break anything.

**Comment by FL33TW00D** [comment]
Sorry commented this out for profiling purposes!

**Comment by FL33TW00D** [comment]
@Huguet57 thanks!
